h2o-core/src/test/java/water/fvec/TestFrameBuilderTest.java

```diff
@@ -0,0 +1,122 @@
+package water.fvec;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.TestUtil;
+
+public class TestFrameBuilderTest extends TestUtil {
+  @BeforeClass
+  public static void setup() { stall_till_cloudsize(1); }
+
+  @Test
+  public void testEmpty(){
+    Frame fr = new TestFrameBuilder().build();
+
+    assert fr.vecs().length == 0;
```

Comment:
Ok, changing all asserts. What's the advantage of using `org.junit.Assert` in simple cases where we just compare 2 values over the regular java assert ? Just to know. Thanks